fmriprep/interfaces/fmap.py

```diff
@@ -68,12 +69,14 @@ def _run_interface(self, runtime):
                     iterations=self.inputs.mask_erode
                     ).astype(np.uint8)  # pylint: disable=no-member
 
-        self._results['out_file'] = genfname(self.inputs.in_file, suffix='enh')
+        self._results['out_file'] = fname_presuffix(
+            self.inputs.in_file, suffix='enh', newpath=os.getcwd())
```

Comment:
`genfname` put a `_` before suffices, so this should be `suffix='_enh'` to keep behavior unchanged.

And since we're in `_run_interface`, we can avoid the syscall with `runtime.cwd`. (I know that this isn't a necessary optimization since we're not bottlenecked by syscalls; it's just a pattern I've been using in my interfaces to use the information provided in `runtime` when possible.)

(Not going to tag all of the uses of `fname_presuffix`, but both comments apply to most.)